tests/unit/test_matrix.py

```diff
@@ -256,3 +256,9 @@ def it_returns_correct_base_values(self, _cube_result_matrix_prop_, prop_name):
     @pytest.fixture
     def _cube_result_matrix_prop_(self, request):
         return property_mock(request, Assembler, "_cube_result_matrix")
+
+
+class Describe_CatXCatMatrix(object):
+    def it_provides_its_unweighted_counts_to_help(self):
+        fake_counts = Mock()
+        assert _CatXCatMatrix(None, None, fake_counts).unweighted_counts == fake_counts
```

Comment:
I would use a literal for the counts, like [1, 2, 3] or something. They could just be inlined here I think, no need for a name, although you could do:

```python
assert _CatXCatMatrix(None, None, unweighted_counts=[1, 2, 3]).unweighted_counts == [1, 2, 3]
```

Reply:
sure, but this probably got fixed by you :)